### PR TITLE
New version: Fix few bugs

### PR DESCRIPTION
**Pull request by Nyavlys**
ACGtk new version. A few bug fix and ocaml 4.12.0 compatibility fix.

### CODE REVIEW

File: packages/acgtk/acgtk.1.5.3/opam [hunk 1]
```diff
@@ -0,0 +1,45 @@
+opam-version: "2.0"
+maintainer: "sylvain.pogodalla@inria.fr"
+
+build: [
+  ["dune" "subst"] {dev}
+# remove the -p to also build the local libraries: conflict with the
+# fact that some libraries are also part of the acgtkLib package
```
Comment (mseri) [inline packages/acgtk/acgtk.1.5.3/opam:7]: Does this require to add a `conflicts: ["acgtklib"]` section?

Comment (Nyavlys) [reply packages/acgtk/acgtk.1.5.3/opam:7]: I don't think so. The acgtkLib package was only meant for local dev usage and is not meant to be distributed with opam anyway, but I never dig deeply enough in opam+dune to clearly understand what the problem was. Anyway, it is not used anymore.

File: packages/acgtk/acgtk.1.5.3/opam [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+install: ["dune" "install"]
+
+depends: [
+  "ocaml" { >= "4.05.0" }
+  "dune" { >= "1.4" }
+  "menhir"
+  "ANSITerminal"
+  "fmt"
+  "logs"
+  "mtime"
```
Comment (mseri) [inline packages/acgtk/acgtk.1.5.3/opam:21]:
```suggestion
  "mtime" {>= "1.0.0"}
```
This is where `mtime.clock.os` was introduced

File: packages/acgtk/acgtk.1.5.3/opam [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+
+depends: [
+  "ocaml" { >= "4.05.0" }
+  "dune" { >= "1.4" }
+  "menhir" {>= "20181113"}
+  "ANSITerminal"
+  "fmt"
+  "logs"
+  "mtime" {>= "1.0.0"}
+  "cmdliner"
```
Comment (mseri) [inline packages/acgtk/acgtk.1.5.3/opam:22]:
```suggestion
  "cmdliner" {>= "1.0.0"}
```
This is wheer `Manpage.s_description` should have been introduced

**Comment by mseri** [comment]
Looks good now, sorry for the delay

**Pull request merged**